### PR TITLE
Remove redundant code and update tls method

### PR DESCRIPTION
**Pull request by realsdx**

### CODE REVIEW

File: heralding/capabilities/rdp.py [hunk 1]
```diff
@@ -68,122 +54,102 @@ class RDP(HandlerBase):
...
-        cc_pdu = cc_pdu_obj.getFullPacket()
-        writer.write(cc_pdu)
-        await writer.drain()
-        if cc_pdu_obj.sentNegoFail:
-            logger.debug("Sent x224 RDP Negotiation Failure PDU")
-            session.end_session()
-            return
-        logger.debug("Sent x244CLinetConnectionConfirm PDU")
+        try:
+            data = await reader.read(2048)
```
Comment (johnnykv) [inline heralding/capabilities/rdp.py]: Why don’t you use recv_next_tpkt here?

File: heralding/capabilities/rdp.py [hunk 2]
```diff
@@ -68,122 +54,102 @@ class RDP(HandlerBase):
...
-            channel_init = channel_req.initiator
-            channel_cnf = MCSChannelJoinConfirmPDU(
-                channel_init, channel_id).getFullPacket()
-
-            await self.send_data(writer, channel_cnf, tls_obj)
+
+            # in tls attach user request and erect domain are not merged together
             logger.debug(
-                "Sent: MCS Channel Join Confirm of channel %s" % (channel_id))
+                "Received: ErectDomainRequest and AttactUserRequest(not in tls) : "+repr(data))
```
Comment (johnnykv) [inline heralding/capabilities/rdp.py]: Do we still need that “not in tls” comment?

File: heralding/capabilities/rdp.py [hunk 3]
```diff
@@ -68,122 +54,102 @@ class RDP(HandlerBase):
...
-            if v < 0:
-                break
-            channel_id = channel_req.channelID
-            channel_init = channel_req.initiator
-            channel_cnf = MCSChannelJoinConfirmPDU(
-                channel_init, channel_id).getFullPacket()
-
-            await self.send_data(writer, channel_cnf, tls_obj)
+
+            # in tls attach user request and erect domain are not merged together
```
Comment (johnnykv) [inline heralding/capabilities/rdp.py]: Is this comment still relevant?

File: heralding/capabilities/rdp.py (hunk 1 above)
Comment (realsdx) [reply heralding/capabilities/rdp.py]: earlier, i did that because it is non-tls packet. but now it should change

**Comment by johnnykv** [comment]
#25

**Comment by realsdx** [comment]
Speed improvement is quite good. That's why i did it.

**Pull request merged**